### DISCUSSION TITLE
No working anymore

### DISCUSSION THREAD

**Discussion by Hunter321noooob** [mmagnus/Pocket-Plus-Calibre-Plugin#57] [Q&A] [answered]
I just discovered this project only some month ago unfortunally, now i get error "UnboundLocalError: local variable 'response' referenced before assignment", seems is not working anymore. I am the only ? How to solve this? I know nothing about programming, sorry

**Comment by rga5321** [comment, marked as answer]
Might it me #56 ?

Try lowering the value of MAX_ARTICLES_PER_FEED

**Reply by rga5321** [reply]
@Hunter321noooob download the latest version and try again, I think it's fixed now.

**Reply by mmagnus** [reply]
It works even with my own old version. I think the Pocket fixed something on the own side.

**Reply by Hunter321noooob** [reply]
> @Hunter321noooob download the latest version and try again, I think it's fixed now.

Thanks

**Comment by mmagnus** [comment]
It works even with my own old version. I think the Pocket fixed something on the own side.

**Reply by Hunter321noooob** [reply]
Thanks, I lowered the feed number from 1000 to 50, and it works, when I finish reading the downloaded articles in a few weeks, I'll try putting 1000 again out of curiosity.
